http/server.go

```diff
@@ -13,9 +13,9 @@ import (
 	"time"
 
 	"github.com/drand/drand/chain"
+	clientinterface "github.com/drand/drand/client/interface"
```

Comment:
:/

Reply:
unhappy with the name/location of this, or something else?